test/ClientTest.cpp

```diff
@@ -637,6 +643,53 @@ single_conn_sql_statements(Connector<BUFFER, NetProvider> &client)
 	
 	printResponse<BUFFER, NetProvider>(conn, *response);
 
+
+	TEST_CASE("prepare");
+	rid_t pr_select = conn.prepare("SELECT * FROM testing_sql;");
+	client.wait(conn, pr_select, WAIT_TIMEOUT);
+	fail_unless(conn.futureIsReady(pr_select));
+	response = conn.getResponse(pr_select);
+	fail_unless(response != std::nullopt);
+	fail_unless(response->body.stmt_id != std::nullopt);
```

Comment:
You forgot to update schema script: at the moment there's still no testing_sql space, so this test must fail on fresh build.